frameworks/moveos-stdlib/sources/module_store.move

```diff
@@ -137,38 +136,7 @@ module moveos_std::module_store {
         let package = borrow_package(package_obj_id);
         object::contains_field(package, name)
     }
-
-    /// Publish modules to the account's storage
-    public fun publish_modules(module_store: &mut Object<ModuleStore>, account: &signer, modules: vector<MoveModule>) {
-        let account_address = signer::address_of(account);
-        if (features::module_publishing_allowlist_enabled()) {
-            ensure_package_id_in_allowlist(account_address);
-        };
-        
-        let upgrade_flag = publish_modules_internal(module_store, account_address, modules);
-        // Store ModuleUpgradeFlag in tx_context which will be fetched in VM in Rust, 
-        // and then announce to the VM that the code loading cache should be considered outdated. 
-        tx_context::set_module_upgrade_flag(upgrade_flag);
-    }
-   
-    // Deprecated
-    // TODO: clean this function when reset genesis
-    /// Entry function to publish modules to the account's storage
-    /// The order of modules must be sorted by dependency order.
-    public entry fun publish_modules_entry(account: &signer, modules: vector<vector<u8>>) {
```

Comment:
Or keep this function for compatibility with the js SDK.

Reply:
If we do not update the SDK at the same time. Maybe the SDK does not call the publish module directly.